roles/network_plugin/calico/tasks/install.yml

```diff
@@ -110,10 +110,23 @@
     - calico_pool_cidr_ipv6 is defined
     - enable_dual_stack_networks
 
+- name: Calico | Remove unwanted annotations and creationTimestamp keys from metadata in Calico manifests
+  # noqa 303 - sed avoids using nested loop
+  shell: >-
+    sed -r -i
+    -e '/^\s{2,4}creationTimestamp: null$/d'
+    -e '/^\s{2,4}annotations:/{:1;/\(devel\)$/!{N;b 1}; /.*/d}'
```

Comment:
extended regexp is required for readability as expressions contain braces and parentheses (I have changed it to `-E` for portability just in case).
shell module is required because command module splits arguments and wildcard is not expanded without a shell

Reply:
My bad, in my head `-r` is recursive, like with grep, and I'm used to `-E` for extended regexp, and indeed I forgot about the wildcard